src/components/Banner/tests/Banner.test.tsx

```diff
@@ -0,0 +1,197 @@
+                                        </Banner>);
+            expect(bannerWrapper.prop('status')).toBe('success');
+        });
+
+        it('should verify banner status property is not defined', () => {
+            const bannerWrapper = mount(<Banner>
+                                        <p>This order was archived on March 7.</p>
+                                        </Banner>);
+            expect(bannerWrapper.prop('title')).toBeUndefined();
+        });
+
+        it('should verify banner icon property', () => {
+            const bannerWrapper = mount(<Banner status="success" icon="arrowDown">
+                                        <p>This order was archived on March 7.</p> 
+                                        </Banner>);
+            expect(bannerWrapper.prop('icon')).toBe('arrowDown');
+        });
+
+        it('should verify banner icon property is not defined', () => {
+            const bannerWrapper = mount(<Banner>
+                                        <p>This order was archived on March 7.</p>
+                                        </Banner>);
+            expect(bannerWrapper.prop('title')).toBeUndefined();
+        });
+
+        it('should verify banner children property', () => {
+            const bannerWrapper = mount(<Banner
+                                        title="Some of your product variants are missing weights"
+                                        status="warning"
+                                        action={{content: 'Edit variant weights'}}>
+                                        <p>Add weights to show accurate rates.</p>
+                                        </Banner>);
+            expect(bannerWrapper.prop('children').props.children).toBe('Add weights to show accurate rates.');
+        });
+
+        it('should verify banner children property is not defined', () => {
+            const bannerWrapper = mount(<Banner/>);
+            expect(bannerWrapper.prop('children')).toBeUndefined();
+        });
+    });
+
+    describe('Verify Banner component functions called', () => {
+        it('should verify banner action is called', () => {
+            const spy = jest.fn();
+            const action: Action = {content: 'Action Content', onAction: () => { spy(); } };
+            const bannerWrapper = mount(<Banner title="Order archived" status="success" action={action}>
+                                        <p>This order was archived on March 7, 2017 at 3:12pm EDT.</p> 
+                                        </Banner>);
+            bannerWrapper.find('button').simulate('click');
+            expect(spy).toBeCalled();
+        });
+
+        it('should verify banner secondaryAction is called', () => {
+            const spy = jest.fn();
+            const secondaryAction: Action = {content: 'Action Content', onAction: () => { spy(); } };
+            const bannerWrapper = mount(<Banner title="Order archived" status="success" action={secondaryAction}>
```

Comment:
I think secondaryAction needs to be pass to secondaryAction property not action proeprty. Also, test if both actions are passed and both actions onAction method are called.